app/views/docs/functions.phtml

```diff
@@ -325,6 +325,8 @@ func main(req: RequestValue, res: RequestResponse) throws -> RequestResponse {
     return res.json(data: task)
 }</code></pre>
         </div>
+        <b>Function Structure</b>
+        <p>When using Swift, it doesn't matter in which file you define your main function, but there has to be a file ending with <b>.swift</b> that defined a <b>main</b> method. Be aware, you can only have one method called 'main' across your whole function.</p>
```

Comment:
```suggestion
        <p>With Swift, your <b>entrypoint</b> can be empty since Appwrite automatically infers it from the location of your <b>main()</b> function. Just ensure that your cloud function has a <b>single declaration of main()</b> across your source files.</p>
```